### PR TITLE
Disable collect fee

### PR DESCRIPTION
**Pull request by XiaoYhun**

### CODE REVIEW

File: src/components/YieldPools/ListItem.tsx [hunk 1]
```diff
@@ -74,7 +74,7 @@ const ListItem = ({ farm, setSharedPoolAddress }: ListItemProps) => {
   const currentTimestamp = Math.floor(Date.now() / 1000)
 
   const qs = useParsedQueryString()
-  const tab = qs.tab || 'active'
+  const type = qs.type || 'active'
```
Comment (namgold) [inline src/components/YieldPools/ListItem.tsx:77]: For minimal change code :tien-sweating:
```suggestion
  const tab = qs.type || 'active'
```

File: src/components/YieldPools/ListItem.tsx [hunk 2]
```diff
@@ -628,13 +628,13 @@ const ListItem = ({ farm, setSharedPoolAddress }: ListItemProps) => {
 
             <Flex grid-area="action" justifyContent="flex-end" sx={{ gap: '4px' }}>
               <ActionButton
-                disabled={tab === 'ended'}
+                disabled={type === 'ended'}
```
Comment (namgold) [inline src/components/YieldPools/ListItem.tsx:631]:
```suggestion
                disabled={tab === 'ended'}
```

File: src/components/YieldPools/ListItem.tsx [hunk 3]
```diff
@@ -628,13 +628,13 @@ const ListItem = ({ farm, setSharedPoolAddress }: ListItemProps) => {
...
                 onClick={() => {
                   setModalType('stake')
                 }}
               >
                 <MouseoverTooltip text={t`Stake`} placement="top" width="fit-content">
-                  <Plus color={tab !== 'ended' ? theme.primary : theme.subText} size={16} />
+                  <Plus color={type !== 'ended' ? theme.primary : theme.subText} size={16} />
```
Comment (namgold) [inline src/components/YieldPools/ListItem.tsx:637]:
```suggestion
                  <Plus color={tab !== 'ended' ? theme.primary : theme.subText} size={16} />
```

File: src/pages/ProAmmPool/index.tsx [hunk 4]
```diff
@@ -56,7 +56,17 @@ export default function ProAmmPool() {
...
+        item.map(it => {
+          if (it.userDepositedNFTs.length > 0) {
+            return it.userDepositedNFTs.map(i => {
+              return { ...i, endTime: it.endTime }
+            })
+          } else {
+            return []
+          }
+        }),
+      )
```
[the remark this answers is not in the document]
Comment (XiaoYhun) [reply src/pages/ProAmmPool/index.tsx:69]: :)))) any better way?

File: src/components/YieldPools/ListItem.tsx (hunk 1 above)
Comment (XiaoYhun) [reply src/components/YieldPools/ListItem.tsx:77]: :)))))

File: src/pages/ProAmmPool/index.tsx (hunk 4 above)
[the remark this answers is not in the document]
Comment (hungdoansy) [reply src/pages/ProAmmPool/index.tsx:69]: :tien-facepalm:

[the remark this answers is not in the document]
Comment (namgold) [reply src/pages/ProAmmPool/index.tsx:69]: no =))

[the remark this answers is not in the document]
Comment (nguyenhoaidanh) [reply src/pages/ProAmmPool/index.tsx:69]: shorter code:         `.flat().flat()` => `.flat(2)`

**Review by nguyenhoaidanh — APPROVED** [review]

**Review by hungdoansy — APPROVED** [review]

**Pull request merged**